mat64/io_test.go

```diff
@@ -39,3 +49,36 @@ func TestDenseRW(t *testing.T) {
 		}
 	}
 }
+
+func TestVectorRW(t *testing.T) {
+	for i, test := range []*Vector{
+		NewVector(0, []float64{}),
```

Comment:
Then you'll need to put in a check for `n < 0` because otherwise the `Data: make([]float64, v.n),` line will panic.